notebooks/MAST/HSC/CASJOBS/HCV_CASJOBS/HCV_casjobs_demo.ipynb

```diff
@@ -0,0 +1,995 @@
+    "\n",
+    "x = nova_814['MJD']\n",
+    "y = nova_814['CorrMag']\n",
+    "e = nova_814['MagErr']\n",
+    "plt.errorbar(x, y, yerr=e, fmt='or', ecolor='k', elinewidth=1, markersize=8, label='ACS F814W')\n",
+    "\n",
+    "plt.gca().invert_yaxis()\n",
+    "plt.xlabel('MJD [days]')\n",
+    "plt.ylabel('magnitude')\n",
+    "plt.legend(loc='best', title=f'Nova in M87 (MatchID: {matchid})')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get HLA image cutouts for the nova <a name=\"cutouts\"></a>\n",
+    "\n",
+    "The [Hubble Legacy Archive (HLA)](https://hla.stsci.edu) images were the source of the measurements in the HSC and HCV, and it can be useful to look at the images.  Examination of the images can be useful to identified cosmic-ray contamination and other possible image artifacts.  In this case, no issues are seen, so the light curve is reliable.\n",
+    "\n",
+    "Note that the ACS F606W images of M87 have only a single exposure, so they do have cosmic ray contamination.  The accompanying F814W images have multiple exposures, allowing CRs to be removed.  In this case the F814W combined image is used to find objects, while the F606W exposure is used only for photometry.  That reduces the effects of F606W CRs on the catalog but it is still a good idea to confirm the quality of the images.\n",
+    "\n",
+    "The `get_hla_cutout` function reads a single cutout image (as a JPEG grayscale image) and returns a PIL image object.  See the documentation on the [fitscut image cutout service](http://hla.stsci.edu/fitscutcgi_interface.html) for more information on the web service being used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_hla_cutout(imagename, ra, dec, size=33, autoscale=99.5, asinh=True, zoom=1):\n",
+    "    \n",
+    "    \"\"\"Get JPEG cutout for an image\"\"\"\n",
+    "    \n",
+    "    url = \"https://hla.stsci.edu/cgi-bin/fitscut.cgi\"\n",
+    "    r = requests.get(url, params=dict(ra=ra, dec=dec, size=size, format=\"jpeg\",\n",
+    "                                      red=imagename, autoscale=autoscale, asinh=asinh, zoom=zoom))\n",
+    "    im = Image.open(BytesIO(r.content))\n",
+    "    return im\n",
+    "\n",
+    "\n",
+    "# sort images by magnitude from brightest to faintest\n",
+    "phot = nova_606\n",
+    "isort = np.argsort(phot['CorrMag'])\n",
+    "# select the brightest, median and faintest magnitudes\n",
+    "ind = [isort[0], isort[len(isort)//2], isort[-1]]\n",
+    "\n",
+    "# we plot zoomed-in and zoomed-out views side-by-side for each selected image\n",
+    "nim = len(ind)*2\n",
+    "ncols = 2 # images per row\n",
+    "nrows = (nim+ncols-1)//ncols\n",
+    "\n",
+    "imsize1 = 19\n",
+    "imsize2 = 101\n",
+    "mra = nova_tab['RA'][0]\n",
+    "mdec = nova_tab['Dec'][0]\n",
+    "\n",
+    "plt.rcParams.update({\"font.size\": 16})\n",
```

Comment:
Not clear on why this is needed gien the universal setting above?

Reply:
I must have overlooked it or it didn't meet my search criteria

Reply:
Deleted